index.html

```diff
@@ -6,7 +6,7 @@
     <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
     <script src="script.js"></script>
 </head>
-<body>
+<body onload="quoteToMovie()">
```

Comment:
should we change this to a jquery document.ready method to be consistent with our other methods